ArduCopter/Copter.h

```diff
@@ -796,6 +796,7 @@ class Copter : public AP_Vehicle {
     // mode.cpp
     bool set_mode(Mode::Number mode, ModeReason reason);
     bool set_mode(const uint8_t new_mode, const ModeReason reason) override;
+    uint8_t get_mode() const override { return (uint8_t)control_mode; }
```

Comment:
we discussed on the call that we should probably make this use the control->mode_number() like the other vehicles (if possible).

Reply:
I'm done. #13431